test/test.js

```diff
@@ -3,8 +3,9 @@ var child = require('child_process');
 var exec = child.exec;
 var spawn = child.spawn;
 var fs = require('fs');
-var suitcss = require('../lib/index.js');
+var suitcss = require('../lib');
 var path = require('path');
+var config = require('../postcss.config.js');
 
 var features = [
```

Comment:
I was thinking a better way to solve this is to have one input file with everything in it (custom media, vars etc) and then just test the output. This would ensure all the plugins worked together. 

For instance it's important that [`custom-properties` go before `calc`](https://github.com/cssnext/cssnext/blob/master/src/features.js) and testing them in isolation won't catch errors like that.